src/tasks/harbor/harbor.ts

```diff
@@ -2,13 +2,18 @@ import {
   ConfigureApi,
   HttpBearerAuth,
   MemberApi,
+  // eslint-disable-next-line no-unused-vars
```

Comment:
can we just put this to the top of the file? repeating the same exclusion so many times is unnecessary

Reply:
and why not remove unused vars?

Reply:
I you search through the file you will find out that it is used.
Then linter may have a bug

Reply:
I think @Morriz means you can add a global eslint flag in the first line of the files, then it gets applied to all lines.

Reply:
ah, noe we do not want to disable this check for the whole file